### ISSUE TITLE
您好！我注意到您这篇文章在MVSA-Single数据集上也进行了相关实验，我想知道调整哪些实验参数才能够复现出论文中的效果

### ISSUE THREAD

**Issue by wang819959** [ZixianGao/EUAR#1]

**Comment by ZixianGao** [comment]
可以具体描述一下问题吗，MVSA-Single上的效果（Acc:79.58）应该不难复现。可以参考这篇basecode [https://github.com/QingyangZhang/QMF]的做法.

**Comment by wang819959** [comment]
就是除了模型整体方法需要将三模态改为双模态之外，还需要改变哪些参数，比如学习率、global_configs中，视觉模态维度需要变化嘛？我看到论文里面有描述说在MVSA-Single数据集上采用的是ResNet-152 作为视觉特征提取器。那您这篇的参数和[https://github.com/QingyangZhang/QMF]的参数一致嘛？

**Comment by ZixianGao** [comment]
有一些改动，但正常来说MVSA-Single应该是比较容易跑到这个值的，我在做实验的时候最高甚至能到81、82。你可以先按照QMF里的试试，库里也有提供数据集的百度网盘链接。视觉提取是ResNet-152，文本提取是Bert。

**Comment by wang819959** [comment]
你好，不知道为什么我现在跑出来的实验效果只有(ACC:72.89)?不知是否和这些参数有关？如果您能提供给我一些帮助，我将非常感谢！
<img width="610" alt="b9a812d3d9dbe6eba53c03feef0821f2" src="https://github.com/user-attachments/assets/54cf5d45-07f0-4171-8fdc-0ccf1e4d417d">


<img width="673" alt="77256c85dcfb950212cde2668bc39ebf" src="https://github.com/user-attachments/assets/d12eeb36-7af1-4895-8b20-e370baab8d0f">

**Comment by ZixianGao** [comment]
参数问题应该不会导致效果差这么多。建议您可以先按照我上面说的QMF的codebase先跑一遍。

**Comment by 123sghn** [comment]
> 你好，不知道为什么我现在跑出来的实验效果只有(ACC:72.89)?不知是否和这些参数有关？如果您能提供给我一些帮助，我将非常感谢！ <img alt="b9a812d3d9dbe6eba53c03feef0821f2" width="610" src="https://private-user-images.githubusercontent.com/174856738/382700087-54cf5d45-07f0-4171-8fdc-0ccf1e4d417d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQwMTEzMDAsIm5iZiI6MTc0NDAxMTAwMCwicGF0aCI6Ii8xNzQ4NTY3MzgvMzgyNzAwMDg3LTU0Y2Y1ZDQ1LTA3ZjAtNDE3MS04ZmRjLTBjY2YxZTRkNDE3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwN1QwNzMwMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NGYwMmM5NTE0YmY2ZWNmMmMzMDBjYWUyYmFhZjFkM2UxNTkxZDMxOWJmZTM2ZDZiMDZhNGI0OTlmZGVlNzBiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DibM_6B3VWNMRPQ-dCJYzyajbX_9OV5TPhArrbcxYeU">
> 
> <img alt="77256c85dcfb950212cde2668bc39ebf" width="673" src="https://private-user-images.githubusercontent.com/174856738/382700105-d12eeb36-7af1-4895-8b20-e370baab8d0f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQwMTEzMDAsIm5iZiI6MTc0NDAxMTAwMCwicGF0aCI6Ii8xNzQ4NTY3MzgvMzgyNzAwMTA1LWQxMmVlYjM2LTdhZjEtNDg5NS04YjIwLWUzNzBiYWFiOGQwZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQwN1QwNzMwMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGIyZWQ3MWE5OTFlODhjZjMwYjAwNGU0OTNmMjNhMmIxOWZkNTNmZjhlMjU2MjVhZjFiZmE1NjY0YmZlZTYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DCOr2nmW8tZcbag1Eg1Lpi_6OetHkghS-mk1HQcuPrM">

你好，请问前辈复现出论文中提到的效果了么？

**Comment by ZixianGao** [comment]
这个值应该不难复现，也可以参考我们另一篇文章的做法 [EAU&URMF](https://github.com/CFM-MSG/Code_URMF/tree/main)。

**Comment by 123sghn** [comment]
您好，我想问一下您是否清楚QMF这篇论文中提供的关于MVSA-Single这个数据集的具体原因，因为我注意到这个数据集规模和MVSA-Single的官方数据集规模并不一致。它论文中提到：
`Image-text classification. For FOOD-101, following the previous work (Kiela et al., 2019), there are 60101 image-text pairs in the training set, 5000 image-text pairs in the validation set, and 21695 image-text pairs in the test set. For MVSA, we conduct the division strategy presents in (Kiela et al., 2019). There are 1555 image-text pairs in the training set. The validation set contains 518 image-text pairs, and the test set contains 519 image-text pairs.`
但我在其提到的文章中并没有找到相应的”划分依据“，请问您知道么？

**Comment by ZixianGao** [comment]
我们为了公平比较直接采用了他们提供的数据集，没有深究划分依据。
